crates/mpz-ole-core/src/ole/mod.rs

```diff
@@ -0,0 +1,3 @@
+//! Provides implementations of OLE with errors (OLEe) protocols.
```

Comment:
```suggestion
//! Oblivious linear evaluation protocols
```

The "with errors" should be a caveat documented on the specific implementation, not at the module level

Reply:
While we're at it, we can be more verbose in our comments "oblivious linear evaluation" is also a short-hand which may be confusing to the uninitiated. The full term "oblivious linear **function** evalution", i.e. there exists a linear function which gets evaluated obliviously.